### PR TITLE
XWIKI-13288 : Filtering on wiki pretty name is broken in wikis index

### PR DESCRIPTION
**Pull request by acotiuga**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
       #set($tableAlias = $order)
     #end
     ## only safe version should be used in SQL queries
-    #set($safe_tableAlias = "$!{tableAlias.replaceAll('[^a-zA-Z0-9_.]','')}")
+    #set($safe_tableAlias = $tableAlias)
     #livetable_getTableAlias($safe_tableAlias)
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml]: I think

    #livetable_getTableAlias($tableAlias)

works too, no? You shouldn't have to set the variable before.

Comment (acotiuga) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml]: I need both variables. There is also a check for $tableAlias in it's initial state.
`#elseif($tablelist.contains($tableAlias))`

Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml]: My proposal doesn't remove any variable. Have you tried it?

Comment (acotiuga) [inline xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml]: Yes, it works. Thanks!

**Pull request merged**